Core/src/Logger/AppEngineFlexV2Handler.php

```diff
@@ -0,0 +1,67 @@
+<?php
+/*
+ * Copyright 2019 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+namespace Google\Cloud\Core\Logger;
+
+use Monolog\Formatter\FormatterInterface;
+use Monolog\Handler\StreamHandler;
+use Monolog\Logger;
+
+/**
+ * Monolog 2.x handler for logging on App Engine flexible environment.
+ *
+ * If you are using Monolog 1.x, use AppEngineFlexV2Handler instead.
+ *
+ * @see AppEngineFlexHandler
```

Comment:
`@see` tags are only recognized by our doc generator for full URIs. Could you remove this annotation and instead change line 26 to the following?

```
If you are using Monolog 1.x, use {@see Google\Cloud\Core\Logger\AppEngineFlexV2Handler} instead.
```

Same thing in other classes. Please make sure to wrap the lines before 80 characters if possible, and at absolute most at 120 characters (longer will cause a style check error). Don't wrap within the `{@see}` tag. :)

Reply:
Done!